netDumbster/MimeKitExtensions.cs

```diff
@@ -130,7 +132,7 @@ public static MailMessage ConvertToMailMessage(this MimeKit.MimeMessage message)
             var sender = message.Sender;
 
             foreach (var header in message.Headers)
-                msg.Headers.Add(header.Field, header.Value);
+                msg.Headers.Add(header.Field, string.IsNullOrWhiteSpace(header.Value) ? "<null>" : header.Value);
```

Comment:
Not sure how the standard works here. I would ignore empty headers instead of adding the string `<null>`as the value. Something like:

``` csharp
            foreach (var header in message.Headers)
            {
                if (!string.IsNullOrEmpty(header.Value))
                {
                    msg.Headers.Add(header.Field, header.Value);
                }
            }
```